.github/workflows/docker-manual.yaml

```diff
@@ -89,15 +85,6 @@ jobs:
                 }
               else
                 empty
-              end,
-              if ${{ inputs.push-ee-to-ecr }} then
-                {
-                  "name": "tenzir-ee", "target": "tenzir-ee", "registries": [
-                   "622024652768.dkr.ecr.eu-west-1.amazonaws.com"
```

Comment:
@tobim doesn't this break things? We don't push the other image to ECR.

Reply:
This file is only used for the manual dispatched workflow. The only image that is automatically pushed to ECR is `tenzir-demo`, and that is unchanged by this PR.